Tella/Scenes/Uwazi/ViewModel/SubmittedEntityViewModel.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  SubmittedEntityViewModel.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 10/5/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class SubmittedEntityViewModel: ObservableObject {
+    
+    var mainAppModel : MainAppModel
+    var entityInstance: UwaziEntityInstance?
+    var uwaziSubmissionViewModel: UwaziSubmissionViewModel?
+    private var subscribers = Set<AnyCancellable>()
+    var uwaziVaultFiles : [UwaziVaultFile] = []
+    @Published var shouldShowToast : Bool = false
+    @Published var toastMessage : String = ""
+
+    var entityTitle : String {
+        guard let stringValue = self.entityInstance?.title else { return "" }
+        return stringValue
+    }
+    
+    var templateName : String {
+        String(format: "%@ %@", LocalizableUwazi.uwaziEntitySummaryDetailTemplateTitle.localized, entityInstance?.collectedTemplate?.entityRow?.name ?? "")
+    }
+    
```

Comment:
Let's use string interpolation instead of String(format ..) I think it's more clear -> ```
var templateName: String {
    let templateTitle = LocalizableUwazi.uwaziEntitySummaryDetailTemplateTitle.localized
    let entityName = entityInstance?.collectedTemplate?.entityRow?.name ?? ""
    return "\(templateTitle) \(entityName)"
}
```